locale/ga_IE

```diff
@@ -0,0 +1 @@
+ga
```

Comment:
It's a symlink. We're doing the same for `fy-NL` -> `fy` so I thought it'd be acceptable here.